src/cgame/cg_consolecmds.cpp

```diff
@@ -490,6 +490,7 @@ static const struct
 	{ "notarget",         0,                       0                },
 	{ "prevframe",        CG_TestModelPrevFrame_f, 0                },
 	{ "prevskin",         CG_TestModelPrevSkin_f,  0                },
+	{ "putcon",           0,                       0                },
```

Comment:
command completion is the 3rd column here

Reply:
This would require me to investigate how to write the functions, though, and that's a rabbit hole I'd rather not go in for now, especially considering the fact I have _zero_ knowledge on the cgame side.